Ensure and document OpenVDB support (#475)
Ekumen-OS/beluga: DEVELOPING.md

    rosdep install --from-path src
    ```

> [!IMPORTANT]

> > `beluga_vdb` requires OpenVDB to be installed in order to be used. For Ubuntu distributions previous to `Noble` OpenVDB needs to be installed from sources before building. Installation instructions can be found [here](beluga_vdb/README.md). If you don't need `beluga_vdb`, you can skip it using `colcon build --symlink-install --packages-ignore beluga_vdb`

Author: fixed on 73d7a934a2d71b073c4c0c02197f74f340bc808a

> 

Author: > our development docker now provides a from-source installation of OpenVDB, which we use regardless of the Ubntu version.

Not quite right, only the `humble` development docker provides a from-source installation of OpenVDB. 
From `Jazzy` OpenVDB dependency is installed using `rosdep`.
For `Noetic` OpenVDB should be installed from sources by the user and it is not provided by the development docker. We havent tested the code for this distro nor OpenVDB.

>  the instructions let you know that you should add OpenVDB if you use beluga_vdb somewhere else 

Right.

>  because we do not declare that dependency in package.xml, and therefore it won't be installed by rosdep

Not quite right, the dependency is declared [here](https://github.com/Ekumen-OS/beluga/blob/ac50ee5b3d42c7462b0103e05737cde4c7f04d23/beluga_vdb/package.xml#L21) but it is broken for `Jammy `  (the binaries files do not include the `Find*.cmake` files). And for `Noble` I think the problem was that the installation path was not the standard one.

    rosdep install --from-path src
    ```

> 🔸`beluga_vdb` requires OpenVDB to be installed in order to be used. For Ubuntu distributions previous to `Noble` OpenVDB needs to be installed from sources before building, installation instructions can be found [here](beluga_vdb/README.md). If you don't need to use `beluga_vdb` you can skip this package using `colcon build --symlink-install --packages-ignore beluga_vdb`

Reviewer: @pvela2017 meta nit: instead of 🔸, consider using [GitHub Markdown alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).

Reviewer: Also, consider:
```suggestion
🔸`beluga_vdb` requires OpenVDB to be installed in order to be used. For Ubuntu distributions previous to `Noble` OpenVDB needs to be installed from sources before building. Installation instructions can be found [here](beluga_vdb/README.md). If you don't need `beluga_vdb`, you can skip it using `colcon build --symlink-install --packages-ignore beluga_vdb`
```

Author: Changed on [5cdc016](https://github.com/Ekumen-OS/beluga/pull/475/commits/5cdc0169d3c9306da9bab59b1b4cb600f7fe9489)
